internal/seclang/directives.go

```diff
@@ -992,6 +992,31 @@ func directiveSecDataset(options *DirectiveOptions) error {
 	return nil
 }
 
+// Description: Configures the maximum number of ARGS that will be accepted for processing.
+// Default: 1000
+// Syntax: SecArgumentsLimit [LIMIT]
+// ---
+// Exceeding the limit will set the REQBODY_ERROR variable,
```

Comment:
I'm not sure that this part of the description is consistent with the implementation. Seems like we are currently logging a warning, but not updating the `REQBODY_ERROR` variable. Maybe we have to generate an error and just directly call `tx.generateRequestBodyError(err)`/`tx.generateResponseBodyError(err)`.
If we want to enforce the rule `200002` (that is part of `coraza.conf-recommended`) we have to set it:
```
SecRule REQBODY_ERROR "!@eq 0" \
"id:'200002', phase:2,t:none,log,deny,status:400,msg:'Failed to parse request body.',logdata:'%{reqbody_error_msg}',severity:2"
```

Reply:
I deleted "set the REQBODY_ERROR variable" in doc. I think if user was exceed limit , we just logged the error.Updating the `REQBODY_ERROR ` is no helpful for user.